### PR TITLE
Obj_visiblock

### PR DESCRIPTION
**Pull request by Wimmerrad**
Object for the Lens of Truth Platforms

### CODE REVIEW

File: assets/xml/objects/object_visiblock.xml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ﻿<Root>
+    <!-- Object for Lens of Truth Platform -->
     <File Name="object_visiblock" Segment="6">
-        <DList Name="object_visiblock_DL_000140" Offset="0x140" />
-        <DList Name="object_visiblock_DL_0001E8" Offset="0x1E8" />
-        <Texture Name="object_visiblock_Tex_0001F0" OutName="tex_0001F0" Format="rgba16" Width="32" Height="32" Offset="0x1F0" />
-        <Collision Name="object_visiblock_Colheader_000AD0" Offset="0xAD0" />
+        <DList Name="gLensOfTruthPlatformOnDL" Offset="0x140" />
+        <DList Name="gLensOfTruthPlatformOffDL" Offset="0x1E8" />
```
Comment (engineer124) [inline assets/xml/objects/object_visiblock.xml:5]:
```suggestion
        <DList Name="gLensOfTruthPlatformDL" Offset="0x140" />
        <DList Name="gLensOfTruthPlatformEmptyDL" Offset="0x1E8" />
```
`gLensOfTruthPlatformOffDL` isn't what's drawn when the platform is off, it's just an empty Display List that's fairly common. So this is the naming I'd recommend

File: assets/xml/objects/object_visiblock.xml [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+        <Texture Name="gLensOfTruthPlatformTex" OutName="LOTPlatform" Format="rgba16" Width="32" Height="32" Offset="0x1F0" />
```
Comment (engineer124) [inline assets/xml/objects/object_visiblock.xml:6]:
```suggestion
        <Texture Name="gLensOfTruthPlatformTex" OutName="lens_of_truth_platform" Format="rgba16" Width="32" Height="32" Offset="0x1F0" />
```
The styling for `OutName` is the texture name in lower case, snake case, without the prefix `g` and without the suffix `Tex`

File: assets/xml/objects/object_visiblock.xml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ﻿<Root>
+    <!-- Object for Lens of Truth Platform -->
```
Comment (engineer124) [inline assets/xml/objects/object_visiblock.xml:2]: Something to take in consideration that we are trying to get everybody who has contributed / wants to contribute to MM decomp to agree to licensing the repo before merging PRs from people who has not yet agreed to it. It is better explained here: https://github.com/louist103/mm-licensing/issues/1.
Could you please read about it and post in that issue if you agree or not. Thanks!

Comment (Wimmerrad) [reply assets/xml/objects/object_visiblock.xml:2]: I'll make sure to agree, thanks for the info and the help

**Review by AngheloAlf — APPROVED** [review]

**Review by engineer124 — APPROVED** [review]

**Review by hensldm — APPROVED** [review]

**Pull request merged**